content/components/page-header.mdx

```diff
@@ -0,0 +1,295 @@
+    <Text as="h3" fontSize={2} m={0}>
+      Default title variant
+    </Text>
+    <Text as="p" mt="0">
+        The default way to let users navigate up in the hierarchy on narrow viewports.
+    </Text>
+  </Box>
+  <img
+    src="https://user-images.githubusercontent.com/586552/222194467-1031b33b-81e7-429b-9201-d154fd7e9d59.png"
+    role="presentation"
+    width="456"
+  />
+</Box>
+
+<Box
+  as="figure"
+  display="flex"
+  alignItems="flex-start"
+  justifyContent="space-between"
+  flexDirection={['column-reverse', 'column-reverse', 'column-reverse', 'column-reverse', 'row']}
+  m={0}
+  mb={3}
+  sx={{gap: 3}}
+>
+  <Box as="figcaption" mt="1">
+    <Text as="h3" fontSize={2} m={0}>
+      Large title variant
+    </Text>
+    <Text as="p" mt="0">
+        Use large for for user-generated content such as issues, pull requests, or discussions.
+    </Text>
+  </Box>
+  <img
+    src="https://user-images.githubusercontent.com/586552/222194464-6c477db6-bb11-445e-b27e-41a0c7267758.png"
+    role="presentation"
+    width="456"
+  />
+</Box>
+
+<Box
+  as="figure"
+  display="flex"
+  alignItems="flex-start"
+  justifyContent="space-between"
+  flexDirection={['column-reverse', 'column-reverse', 'column-reverse', 'column-reverse', 'row']}
+  m={0}
+  mb={3}
+  sx={{gap: 3}}
+>
+  <Box as="figcaption" mt="1">
+    <Text as="h3" fontSize={2} m={0}>
+      Subtitle variant
+    </Text>
+    <Text as="p" mt="0">
+        Use subtitle when a title is already present in the page, such as in a Split page layout.
+    </Text>
+  </Box>
+  <img
+    src="https://user-images.githubusercontent.com/586552/222215097-79bb0d77-af5c-4199-81a1-65d98a7a0556.png"
+    role="presentation"
+    width="456"
+  />
+</Box>
+
+<Box
+  as="figure"
+  display="flex"
+  alignItems="flex-start"
+  justifyContent="space-between"
+  flexDirection={['column-reverse', 'column-reverse', 'column-reverse', 'column-reverse', 'row']}
+  m={0}
+  mb={3}
+  sx={{gap: 3}}
+>
+  <Box as="figcaption" mt="1">
+    <Text as="h3" fontSize={2} m={0}>
+      Leading visual
+    </Text>
+    <Text as="p" mt="0">
+        Leading visuals are optional and appear at the start of the title. They can be octicons, avatars, and other custom visuals that fit a small area.
+    </Text>
+  </Box>
+  <img
+    src="https://user-images.githubusercontent.com/586552/222194458-922bd3bf-475d-4d76-97a1-bd8cda6af6f5.png"
+    role="presentation"
+    width="456"
+  />
+</Box>
+
+<Box
+  as="figure"
+  display="flex"
+  alignItems="flex-start"
+  justifyContent="space-between"
+  flexDirection={['column-reverse', 'column-reverse', 'column-reverse', 'column-reverse', 'row']}
+  m={0}
+  mb={3}
+  sx={{gap: 3}}
+>
+  <Box as="figcaption" mt="1">
+    <Text as="h3" fontSize={2} m={0}>
+      Trailing visual
+    </Text>
+    <Text as="p" mt="0">
+        Trailing visual and trailing text can display auxiliary information. They're placed at the right of the item, and can denote status, privacy details, etc.
```

Comment:
Should we also add that the leading and trailing visuals remain visible on all viewports?

Reply:
fixed now